addon-test-support/setup-global-a11y-hooks.ts

```diff
@@ -18,7 +17,7 @@ export function setupGlobalA11yHooks(
 ) {
   ['visit', 'click', 'doubleClick', 'tap'].forEach((helperName) => {
     let hook = _registerHook(helperName, 'end', async () => {
-      if (shouldForceAudit() && shouldAudit(helperName, 'end')) {
```

Comment:
IMHO, this should be reverted. See https://github.com/ember-a11y/ember-a11y-testing/pull/203/files#r480213736 for details.

Reply:
Done, as per our discussion.